qtt/algorithms/fitting.py

```diff
@@ -145,6 +159,32 @@ def fermi_linear_fitting_function(xdata, a, b, cc, A, T):
 
 # %%
 
+def fit_add_line_arr(xdata, ydata, trimborder=True):
```

Comment:
This function duplicates code from `fit_addition_line` and I would like to prevent that. Can you use the `fit_add_line_arr` in `fit_addition_line`)?

Also: since `fit_add_line_arr` is a public function, I want a proper name, e.g. `fit_addition_line_array`.